acq4/devices/Pipette/tracker.py

```diff
@@ -539,14 +541,14 @@ def mapErrors(
         print("Manipulator missed target %d times" % misses)
 
         filename = self.dev.configFileName("error_map.np")
-        np.save(open(filename, "wb"), self.errorMap)
+        np.save(filename, self.errorMap)
```

Comment:
Are you sure this is the same as `open(..., 'wb')` ?

Reply:
That's what in the numpy source in at least 1.20.2, yeah.